### DISCUSSION TITLE
[Adonis 4.1] Event not emiting

### DISCUSSION THREAD

**Discussion by wemersonrv** [adonisjs/core#1417] [Help] [answered]
Hi folks, i'm facing an issue with events...

first create a listener with `adonis make:listener Message`, and add the code below:
```js
'use strict'

const Message = (exports = module.exports = {})

Message.incoming = async () => {
  console.log('A new message has arrived' )
}
```
After that, i registered the event in `start/event.js`
```js
const Event = use('Event')

Event.on('message::incoming', 'Message.incoming')
```
But when i try to emit the event, nothing happens...
```js
const Event = use('Event')

Event.emit('message::incoming')
```

Did i miss someting?

**Comment by thetutlage** [comment, marked as answer]
Should work, I don't see anything wrong with the code. What happens when you replace the event handler with an inline function?

```js
Event.on('message::incoming',  () => {
  console.log('A new message has arrived' )
})
```

**Reply by wemersonrv** [reply]
> Should work, I don't see anything wrong with the code. What happens when you replace the event handler with an inline function?
> 
> ```js
> Event.on('message::incoming',  () => {
>   console.log('A new message has arrived' )
> })
> ```

I have try this later too. both ways has the same problem: Event listener is not executed.

![image](https://user-images.githubusercontent.com/2028673/89716812-995bd980-d986-11ea-83e7-793cb6485244.png)

**Reply by thetutlage** [reply]
Is `events.js` file even loaded?

**Reply by wemersonrv** [reply]
> Is `events.js` file even loaded?

U kill it! Thanks.

i'm using `start/event.js` but the right is `/start/events.js` ... 

I don't believe it.. a sooo tiny thing makes me lost 2 days!!!!
